q2_feature_table/_summarize/summarize_assets/overview.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="./q2templateassets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./q2templateassets/css/normalize.css">
+    <script src="./app/js/child.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <div class="row">
```

Comment:
There are a lot of `row` nodes in each subpage, is there a top-level bootstrap container which consumes the margin correctly?